nautobot/dcim/api/serializers.py

```diff
@@ -1083,6 +1085,40 @@ def validate(self, data):
         return super().validate(data)
 
 
+class InterfaceSerializerVersion13(InterfaceSerializer, StatusModelSerializerMixin):
```

Comment:
Personally I'd prefer if `InterfaceSerializer` stays the current/latest version of the serializer, and we add `InterfaceSerializerVersion12` as the frozen legacy serializer version.